platforms/quorum/charts/node_constellation/templates/deployment.yaml

```diff
@@ -189,8 +189,15 @@ spec:
           echo slept for 5 to give transaction manager time to start;
 
           args=""
+          NODE_STATUS=""
           if [ $CONSENSUS == 'raft' ]; then
-            args=" --raft --raftport {{ .Values.node.ports.raft }} --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft "
+            NODE_STATUS={{ .Values.node.status }}
+          fi;
+          if [ $CONSENSUS == 'raft' ] && [ $NODE_STATUS == 'new' ]; then
+            args="--raftdnsenable --raft --raftport {{ .Values.node.ports.raft }} --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft --raftjoinexisting {{ .Values.node.peer_id }}"
+          fi;
+          if [ $CONSENSUS == 'raft' ] && [ $NODE_STATUS == 'existing' ]; then
```

Comment:
the node.status should be "new" (default) and "additional" (when new node is added in existing network). Default cannot be "existing" when we create a network from scratch.

Reply:
I will change this. Default and existing are better choices for the variable